worlds/tunic/rules.py

```diff
@@ -326,6 +332,13 @@ def set_location_rules(world: "TunicWorld") -> None:
     set_rule(world.get_location("Hero's Grave - Feathers Relic"),
              lambda state: state.has(laurels, player) and has_ability(prayer, state, world))
 
+    # Bombable Walls
+    for location_name in bomb_walls:
+        # has_sword is there because you can buy bombs in the shop
+        set_rule(world.get_location(location_name), lambda state: state.has(gun, player) or has_sword(state, player))
```

Comment:
Is it worth adding a `has_gun` helper? Or is it used infrequently enough it is just onerous?

Reply:
Nah, has_sword is only there because it can be dependent on options. Gun is always just gonna be `state.has(gun, player)`, so no need to complicate things imo

Reply:
Nice, makes sense 🔥